### ISSUE TITLE
Version documentation through release tags

### ISSUE THREAD

**Issue by andrewfrench** [drud/ddev#1212]
**Is your feature request related to a problem? Please describe.**
Our documentation tracks the repo's master branch, so when a feature has been pushed to master but has not yet been released, it will confusingly appear in documentation but not in any released version of ddev.

**Describe the solution you'd like**
ReadTheDocs supports versioned documentation by tracking tags, which will allow us to effectively stage changes to documentation until features are released publicly. This will also allow users to find documentation corresponding to the version of ddev they're using if it's not the most current version.

**Comment by rfay** [comment]
Yay @andrewfrench - Could you get the keys to the kingdom on this from @tannerjfco and this will be an excellent move. We probably should figure out how to use the company *paid* readthedocs instead of (free?) account we're using.

**Comment by andrewfrench** [comment]
Yes, this seems like a great time to consolidate into one paid account. I tried chasing down the ddev readthedocs instance and just ended up very confused, but it sounds like @tannerjfco's the guy to talk to.
